### DISCUSSION TITLE
Reordering or sorting tables

### DISCUSSION THREAD

**Discussion by cooper-david-a** [mgreminger/EngineeringPaper.xyz#300] [Ideas]
It doesn't seem to be possible to manually re-order or sort data tables or selection tables.  I would find that useful as I build a library of Prebuilt Tables similar to the ones you've created.

As I am writing this, I noticed the import/export from/to .csv.  I guess that is the workaround for data tables.  Is something similar possible for selection tables?

**Comment by mgreminger** [comment]
Yes, the idea is to do the heaving lifting of organizing the data in Excel or something similar and to use the spreadsheet import and CSV export to interface to the external software (the import will work with both Excel files and csv files, export is only csv). 

Yes, I like the idea of adding spreadsheet import and CSV export to selector tables as well. Won't be too hard to add now that it's implemented for Data Tables. I've add #301

**Comment by mgreminger** [comment]
It took a while, but it's now possible to [import selector tables from spreadsheet files](https://youtu.be/wcAXWE_lBVE). They first need to be imported into a data table and the button bar above the data table now has a "Convert to Selector Table" button. Additionally, data tables now support row labels. If the first column of the spreadsheet is non-numeric, it will automatically be used as row labels, which will become the selection labels when converted to a selector table.
